src/test/java/org/opensearch/neuralsearch/processor/NeuralQueryEnricherProcessorIT.java

```diff
@@ -29,20 +29,21 @@ public class NeuralQueryEnricherProcessorIT extends BaseNeuralSearchIT {
     private static final String ingest_pipeline = "nlp-pipeline";
     private static final String TEST_KNN_VECTOR_FIELD_NAME_1 = "test-knn-vector-1";
     private final float[] testVector = createRandomVector(TEST_DIMENSION);
+    private static String modelId;
 
     @Before
     public void setUp() throws Exception {
         super.setUp();
         updateClusterSettings();
-        prepareModel();
+        modelId = prepareModel();
     }
 
     @After
     @SneakyThrows
     public void tearDown() {
         super.tearDown();
         deleteSearchPipeline(search_pipeline);
-        findDeployedModels().forEach(this::deleteModel);
+        deleteModel(modelId);
```

Comment:
do we need methods for deleting all model ids?

Reply:
yeah I am removing it in next revision.